tests/strategies/test_strategy_ev_charger.py

```diff
@@ -159,3 +165,301 @@ def test_if_preferred_charging_power_overrides_bought_energy(ev_charger_strategy
     expected_energy_kWh = preferred_power_kW * slot_length_hours
     actual_energy_kWh = strategy.state.get_available_energy_to_buy_kWh(time_slot)
     assert isclose(actual_energy_kWh, expected_energy_kWh, rel_tol=1e-03)
+
+
+def test_multiple_concurrent_ev_sessions_aggregate_energy():
```

Comment:
NIT: This title is a bit misleading. I would prefer something like:

```suggestion
def test_get_aggregate_energy_return_correct_values():
```